# *******************************************************************************
# Copyright (c) 2026 Contributors to the Eclipse Foundation
#
# See the NOTICE file(s) distributed with this work for additional
# information regarding copyright ownership.
#
# This program and the accompanying materials are made available under the
# terms of the Apache License Version 2.0 which is available at
# https://www.apache.org/licenses/LICENSE-2.0
#
# SPDX-License-Identifier: Apache-2.0
# *******************************************************************************
diff --git a/src/extensions/score_metamodel/BUILD b/src/extensions/score_metamodel/BUILD
--- a/src/extensions/score_metamodel/BUILD
+++ b/src/extensions/score_metamodel/BUILD
@@ -37,4 +37,5 @@
 filegroup(
     name = "metamodel_yaml",
     srcs = ["metamodel.yaml"],
+    visibility = ["//visibility:public"],
 )
